torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12388,7 +12390,7 @@ def ref_pairwise_distance(input1, input2):
                    skips=(
                        # Reference: https://github.com/pytorch/pytorch/issues/66402
                        DecorateInfo(unittest.expectedFailure, "TestUnaryUfuncs", "test_reference_numerics_hard",
-                                    device_type='cpu', dtypes=(torch.complex64,), active_if=not (IS_MACOS or IS_WINDOWS)),
+                                    device_type='cpu', dtypes=(torch.complex64,), active_if=not (IS_JETSON or IS_MACOS or IS_WINDOWS)),
```

Comment:
what is the purpose of this stuff? I thought the launch bounds allowed all kernels to run.

Reply:
in this case the issue tagged in the comments is that normal x86 linux machines fail on this test and it is known so they have set an expected failure for the test for x86 linux. However, since jetsons use arm, the test actually passes through on it, just like on Windows and MAC, and without my addition of that flag, the unit tester will complain that the test failed since it expected a failure and got a success.